src/player.js

```diff
@@ -1659,6 +1686,11 @@ export default class Player extends FakeEventTarget {
           }
         });
       }
+      if (this.config.playback.playAdsWithMSE) {
```

Comment:
need to add to the default-config and docs

Reply:
It suppose to be relevant for all ad plugin which will come